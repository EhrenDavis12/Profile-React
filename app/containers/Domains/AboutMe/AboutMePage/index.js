/*
 * HomePage
 *
 * This is the first thing users see of our App, at the '/' route
 *
 * NOTE: while this component should technically be a stateless functional
 * component (SFC), hot reloading does not currently support SFCs. If hot
 * reloading is not a necessity for you then you can refactor it and remove
 * the linting exception.
 */

import React from 'react';
import HeaderBar from 'components/Domains/SharedKernel/HeaderBar';

/* eslint-disable react/prefer-stateless-function */
export default class AboutMePage extends React.PureComponent {
  render() {
    return (
      <React.Fragment>
        <HeaderBar
          header="Ehren Davis"
          subMessage="Sr. Developer at Hyland .Inc"
        />
      </React.Fragment>
    );
  }
}
