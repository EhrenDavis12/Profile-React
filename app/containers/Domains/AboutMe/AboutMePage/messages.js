/*
 * HomePage Messages
 *
 * This contains all the text for the HomePage container.
 */
import { defineMessages } from 'react-intl';

export const scope = 'app.containers.Domains.AboutMe.AboutMePage';

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'Ehren Davis',
  },
  subMessage: {
    id: `${scope}.subMessage`,
    defaultMessage: 'Sr. Developer at Hyland .Inc!',
  },
});
