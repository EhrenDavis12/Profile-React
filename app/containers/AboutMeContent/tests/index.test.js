import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import AboutMePage from '../index';

describe('<AboutMePage />', () => {
  it('should render the page', () => {
    const renderedComponent = shallow(<AboutMePage />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
