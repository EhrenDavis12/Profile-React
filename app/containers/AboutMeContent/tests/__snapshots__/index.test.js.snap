// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutMePage /> should render the page 1`] = `
<Fragment>
  <SectionBanner
    header="Self Project Portfolio"
    id="Portfolio"
    image="IMAGE_MOCK"
    strength={800}
  >
    <ProfessionalIntro />
  </SectionBanner>
  <SectionBanner
    header="TechnicalSkills"
    id="TechnicalSkills"
    image="IMAGE_MOCK"
    strength={1000}
  >
    <TechnicalSkills />
  </SectionBanner>
  <SectionBanner
    header="Hobbies"
    id="Hobbies"
    image="IMAGE_MOCK"
    strength={1000}
  >
    <Hobbies />
  </SectionBanner>
  <SectionBanner
    header="ProfessionalExperience"
    id="ProfessionalExperience"
    image="IMAGE_MOCK"
    strength={800}
  >
    <ProfessionalExperience />
  </SectionBanner>
  <SectionBanner
    header="Education"
    id="Education"
    image="IMAGE_MOCK"
    strength={1000}
  >
    <Education />
  </SectionBanner>
</Fragment>
`;
